### ISSUE TITLE
CLI installation notices

### ISSUE THREAD

**Issue by Jako** [modxcms/revolution#12564]
### Summary

There are two PHP Notices during CLI install
### Step to reproduce

Install by CLI (maybe during setup routine too)
### Observed behavior

Two notices:

```
Undefined index: primary in /srv/www/.../html/core/xpdo/om/mysql/xpdomanager.class.php on line 462
```

```
Undefined index: unique in /srv/www/.../html/core/xpdo/om/mysql/xpdomanager.class.php on line 462
```
### Expected behavior

No notices.
### Environment

2.4.0, PHP 5.4

**Comment by bezumkin** [comment]
Confirm

**Comment by Alroniks** [comment]
Also confirm. 
```
PHP 5.5.9-1ubuntu4.11 (cli) (built: Jul  2 2015 15:23:08)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
   with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies
```

**Comment by Mark-H** [comment]
those notices sound like it may be related to an index in the model or something? Are these new installs or updates?

**Comment by bezumkin** [comment]
Only new install in a cli mode. On update I don`t see these notices.

**Comment by Jako** [comment]
New installation.

**Comment by pixelchutes** [comment]
Curious if it could be related to `modManagerLog`? I see the following error during an upgrade from 2.3.x > 2.4.0:

> 
> Error updating table for class modManagerLog
> Array
> (
> [0] => 00000
> [1] => 
> [2] => 
> )

Not sure if the index in question has to do with this table or not?

**Comment by pixelchutes** [comment]
Curious if this is fixed by #12622 ...can anyone confirm?
